test/forms/TaxResultSpec.scala

```diff
@@ -188,6 +188,21 @@ class TaxResultSpec extends BaseSpec with AnyWordSpecLike {
     }
   }
 
+  "Calling the moneyFormatterResult function" should {
+
+    "return the big decimal with a value of 12 to 2dp" in {
+      moneyFormatterResult(BigDecimal(12)) mustBe "12.00"
+    }
+
+    "return 0 to 2dp" in {
+      moneyFormatterResult(BigDecimal(0)) mustBe "0.00"
+      }
+
+    "return a 3dp value to 2dp value" in {
+      moneyFormatterResult(123.455) mustBe "123.45"
```

Comment:
Is this correct result you're expecting? (im not sure myself, please check)

The previous `moneyFormat` function uses `RoundingMode.HALF_UP`, can you add some more test cases around the round up/down values to prove it's doing the same thing. 

e.g. a test case for 123.455 (maybe this should return 123.46), 123.454 (maybe this should return 123.45).